frontend/src/pages/gov/Proposals.js

```diff
@@ -223,6 +226,38 @@ export default function Proposals() {
           }}
         />
       ) : null}
+      {proposals.length === 0 ? (
```

Comment:
proposals?.length ===0 and !proposals?.length both are same. But recommended to use !proposals?.length always.